src/components/SourceTabs.js

```diff
@@ -33,13 +32,19 @@ require("./Dropdown.css");
 function getHiddenTabs(sourceTabs, sourceTabEls) {
   sourceTabEls = [].slice.call(sourceTabEls);
   function getTopOffset() {
-    const topOffsets = sourceTabEls.map(t => t.getBoundingClientRect().top);
+    const topOffsets = sourceTabEls.map(t => t &&
+      t.getBoundingClientRect().top);
     return Math.min(...topOffsets);
   }
 
   const tabTopOffset = getTopOffset();
   return sourceTabs.filter((tab, index) => {
-    return sourceTabEls[index].getBoundingClientRect().top > tabTopOffset;
+    const el = sourceTabEls[index];
+    if (!el) {
```

Comment:
I just tried reverting to line 42; however, `sourceTabEls[index]` was undefined, and I got this error message: `Uncaught TypeError: Cannot read property 'getBoundingClientRect' of undefined`. I think we may have to keep this code because `sourceTabEls[index]` could be undefined.